.github/workflows/publish.yml

```diff
@@ -4,13 +4,12 @@ on:
     paths:
       - 'setup.py'
 
-
 jobs:
   Pipeline:
-    if: github.ref == 'refs/heads/main' || contains(github.ref, 'hotfix/')
+    if: github.ref == 'refs/heads/main'
 
     runs-on: ubuntu-22.04
-    container: quintoandar/python-3-7-java
+    container: python:3.7-buster
```

Comment:
Did you ran a test considering the `alpine` or the `slim` image types? They are tinier than the `buster` images, although they may run into errors due to the lack of native libs inside them.